stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,433 @@
+            </div>
+         </div>
+    </div>
+`;
+
+withActions.storyName = 'Actions';
+withActions.parameters = {
+    docs: {
+        storyDescription: 'feed list item with actions'
+    }
+};
+
+export const feedListGroup = () => `
+    <div class="fd-feed-list fd-feed-list--group" aria-label="Feed List Item group example">
+        <div class="fd-feed-list__body">
+            <div
+                class="fd-avatar fd-avatar--s fd-avatar--placeholder sap-icon--person-placeholder fd-feed-list__thumb" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe"
+            ></div> 
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Cruz: </span>
+                    <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus.        
+                </p>
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 21, 2020</small>
+                </div>
+            </div>
+       </div>
+       
+        <div class="fd-feed-list__body">
+            <div 
+                class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list__thumb" 
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe">    
+            </div>
+         
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus.        
+                </p>
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 22, 2020</small>
+                </div>
+            </div>     
+        </div>
+        
+        <div class="fd-feed-list__body">
+            <div
+                class="fd-avatar fd-avatar--s fd-feed-list__thumb"
+                style="background-image: url(http://lorempixel.com/460/620/people/9/)" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe"
+            ></div> 
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Sanchez: </span>
+                    <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus.        
+                </p>
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+       </div>
+    </div>
+</div>
+`;
+
+feedListGroup.storyName = 'Group';
+feedListGroup.parameters = {
+    docs: {
+        storyDescription: 'Group'

```

Comment:
Evidently, feed list items can be displayed in a group. The example below displays a feed list item with a placeholder image and two standard feed list items.